src/styles/page-template/_template.njk

```diff
@@ -14,7 +14,8 @@
 
 {% if pageConfig.cdn or release_version %}
     {# Production #}
-    {% set assetsURL = "https://cdn.ons.gov.uk/sdc/design-system/" + (pageConfig.cdn or release_version) %}
+    {% set cdn_url = pageConfig.cdn.url or "https://cdn.ons.gov.uk/sdc/design-system/" %}
+    {% set assetsURL = cdn_url + (pageConfig.cdn.version or release_version) %}
```

Comment:
This will break if `pageConfig.cdn.url` doesn't have a trailing slash. We should remove the trailing slash from the default cdn url and add a slash before the version. That way, if someone does provide a trailing slash with a cdn url it will carry on working (with a `//` in the url).

Reply:
I did think about that but that they would be more to do with knowing what you need to pass. But I'm happing to prefix with `/` just so that it works either way.

Reply:
Ad discussed, it didn't work with `//` in the URL, therefore I added a small check to add or not add a trailing slash if required.